### ISSUE TITLE
Save timestamp with high score

### ISSUE THREAD

**Issue by BWagener** [performous/performous#822]
### Suggestion

Having a timestamp for each high score is a feature I think would be cool.

### Use case

Being able to see your progress over time when practicing a song is a great motivation.

This kind of relates to https://github.com/performous/performous/issues/11.
A feature like this would also tie well into https://github.com/performous/performous/issues/11, allowing charting average overall score or score by song over time at some point.

**Comment by Baklap4** [comment]
Looks like a really nice feature and indeed ties well into 11!
Charting the scores can maybe be done within the web ui instead of in performous (or both). Web is more easy to realise some charts i think ;P

**Comment by BWagener** [comment]
For visualization I was thinking web as well!

I accidentally linked the same issue twice. I meant to also refer to https://github.com/performous/performous/issues/240, I also found https://github.com/performous/performous/issues/53 which is related.

Another one I found which is probably obsolete is https://github.com/performous/performous/issues/103

**Comment by Baklap4** [comment]
So the current database.xml is an xml file + it also comes with some flaws. We can think this over and just use json for this. Then again we won't get rid of libxmlpp since we also have a xml parser... for singstar discs... last time i heard @Lord-Kamina about this and it was pretty skeptical if someone even uses this format nowadays

**Comment by Lord-Kamina** [comment]
Performance would likely be a trivial difference if any at all, but if we were to switch it to JSON, the API is so much simpler to work with IMO. 

For visualization (I can already see you groaning) but I'd consider bringing in a simple graphing lib. Otherwise, what would you do, HTML5?

**Comment by Lord-Kamina** [comment]
Also, my reasoning for being is skeptical is that most likely anybody who would import Singstar songs most likely already did it by now.

**Comment by Baklap4** [comment]
Yeah for visualizations we probably can use chart.js (a library which is commonly used)
The json api is way easier/cleaner to work with you're correct!

**Comment by twollgam** [comment]
@Baklap4  @BWagener  the issue is closed automagic by github. I am not sure if that is ok cause maybe not everything is included in #857.
